src/QuantumSdk/Sdk/Sdk.targets

```diff
@@ -97,6 +97,7 @@
       <_QscCommandSkipTargetingFlag Condition="'$(ResolvedQSharpOutputType)' == 'QSharpLibrary' And '$(NoEntryPoint)' != 'true'">$(_NewLine)--skip-targeting</_QscCommandSkipTargetingFlag>
       <_QscCommandSkipMonomorphization Condition="'$(ResolvedTargetCapability)' == 'FullComputation' And $(CSharpGeneration) And !$(EnableQirSubmission)">$(_NewLine)--skip-monomorphization</_QscCommandSkipMonomorphization>
       <_QscCommandPerfDataGenerationFlag Condition="$(PerfDataGeneration)">$(_NewLine)--perf$(_NewLineIndent)$(PerfDataOutputPath)</_QscCommandPerfDataGenerationFlag>
+      <_QscCommandCapabilityWarningsFlag Condition="'$(TreatCapabilityWarningsAsErrors)' == 'true'">$(_NewLine)--error 5023,5024,5025,5026,5027,5028</_QscCommandCapabilityWarningsFlag>
```

Comment:
I don't see where `TreatCapabilityWarningsAsErrors` is set, how is it intended to be used?

Reply:
I expect TreatCapabilityWarningsAsErrors to be set by the user in a project so that capability warnings are treated as errors.